### ISSUE TITLE
Add a Spreadsheets app

### ISSUE THREAD

**Issue by INailedIt** [numworks/epsilon#1027]
#### Problem you'd like to fix

Presently, there exists no spreadsheets app. This makes iteration more of a nuisance than on simpler calculators like the CASIO FX-991EX. A python script can be made and used in classroom environments however, writing a script in a exam setting can take more time than it's worth.

### Describe the solution you'd like

Create a spreadsheets app with similar to functionality to those found on CASIO calculators.

**Comment by Will-of-R-Learn** [comment]
I believe that this should be an additional app instead of a core app.

**Comment by Zenith222** [comment]
Can this please be a priority? There are many situations at this point I’d like to use a spreadsheet app. NumWorks seems so modern but lacks functionality that calculators from the 1990s can accomplish.

**Comment by FaisalAhmed123** [comment]
> I believe that this should be an additional app instead of a core app.

I disagree, it will no longer be useable during examinations with this. Not having this put's students at a disadvantage
